src/components/LmsConfigurations/index.jsx

```diff
@@ -142,11 +161,23 @@ class LmsConfigurations extends React.Component {
           <Collapsible
             styling="card"
             className="shadow"
-            title="Canvas"
+            title="Blackboard"
```

Comment:
This was a typo in master. Just tacked on here since I noticed it when I tested something on stage.

Reply:
Good catch.